src/login-page/LoginPage.css

```diff
@@ -0,0 +1,73 @@
+.sm-login-window {
+  max-width: 400px;
+  border: 1px solid black;
+  border-radius: 5px;
+  margin: 2rem auto;
+}
+.sm-login-container h2 {
+  text-align: center;
+  padding: 0.5rem;
+}
+.sm-input-text {
+  display: flex;
+  flex-direction: column;
+}
+.sm-input-text label {
+  margin: 0 1rem;
+  text-align: left;
+}
+.sm-input-text input {
+  padding: 0.5rem;
+  margin: 0 1rem;
+}
+.login-check {
+  display: flex;
+  gap: 1rem;
+  padding: 1rem;
+}
+.login-check div:last-child {
+  margin-left: auto;
+}
+.btn-primary {
+  margin: 1rem;
+  width: 90%;
+  background-color: gray;
+}
+.btn-primary:hover {
+  background-color: rgb(58, 52, 52);
+}
+.outline-primary {
+  margin: 0 1rem;
+  width: 90%;
+  color: gray !important;
+}
+.outline-primary:hover {
+  background-color: rgb(46, 43, 43) !important;
+  color: white !important;
```

Comment:
**Suggestion**: Don't use !**important**

Reply:
- yes u r right but i don't know how to override previous style which is coming from my component library